pydeseq2/utils.py

```diff
@@ -114,7 +120,7 @@ def test_valid_counts(counts_df):
     if counts_df.isna().any().any():
         raise ValueError("NaNs are not allowed in the count matrix.")
     if ~counts_df.apply(
-        lambda s: pd.to_numeric(s, errors="coerce").notnull().all()
+        lambda s: pd.to_numeric(s, errors="coerce").notnull().all()  # type: ignore
```

Comment:
as much as with imports type: ignore is self explanatory here I think a comment should be added on why it is used

Reply:
Indeed. As I am personally not so familiar with `mypy`, @illumination-k could you add a quick comment to explain? Thanks!

Reply:
This is my mistake. I just put this as temporary for ignore, and I forgot to resolve it.

Reply:
Ok, thanks for clarifying :)